Dockerfile

```diff
@@ -8,6 +8,7 @@ ENV DOCKERSOCKET unix:///var/run/docker.sock
 ENV ENDPOINT /
 ENV IMAGE ubuntu
 ENV CONTAINER_NAME ubuntu-test
+ENV PUBLISH_PORT 9292
```

Comment:
Now you're making ddh specific for one project, dont make a default. (default to `PublishAllPorts`). I suggest removing this.